modules/sekindoUMBidAdapter.js

```diff
@@ -1,91 +1,113 @@
-import { getBidRequest } from 'src/utils.js';
-import { config } from 'src/config';
-
-var CONSTANTS = require('src/constants.json');
-var utils = require('src/utils.js');
-var bidfactory = require('src/bidfactory.js');
-var bidmanager = require('src/bidmanager.js');
-var adloader = require('src/adloader.js');
-var adaptermanager = require('src/adaptermanager');
-
-function SekindoUMAdapter() {
-  function _callBids(params) {
-    var bids = params.bids;
-    var bidsCount = bids.length;
-
+import * as utils from 'src/utils';
+//  import {config} from 'src/config';
+import {registerBidder} from 'src/adapters/bidderFactory';
+export const spec = {
+  code: 'sekindoUM',
+  supportedMediaTypes: ['video'],
+  /**
+   * Determines whether or not the given bid request is valid.
+   *
+   * @param {BidRequest} bid The bid params to validate.
+   * @return boolean True if this is a valid bid, and false otherwise.
+   */
+  isBidRequestValid: function(bid) {
+    if (bid.mediaType == 'video') {
```

Comment:
As of #1483, video ad units may be defined with `mediaTypes: {video: {}}` and would be on this `bid` parameter, you'll probably want to check for that as well

Reply:
done